webcompat/static/js/lib/labels.js

```diff
@@ -24,7 +24,16 @@ issues.LabelsView = Backbone.View.extend({
     "click .js-LabelEditorLauncher.is-active": "closeEditor"
   },
   keyboardEvents: {
-    "e": "editLabels"
+    "l": "labelWrap"
+  },
+  labelWrap: function(e){
```

Comment:
nit: can you add a space between `) {` here? jslint is unhappy:

`29:25  error  Missing space before opening brace  space-before-blocks`